src/BinaryTree/BinaryTree.cs

```diff
@@ -33,6 +33,23 @@ public BinaryTree(int capacity)
         public BinaryTree()
         {
         }
+        
+        public void PrettyPrint()
+	    {
+		    PrettyPrintRec(this._head, 0);
+	    }
+
+	    private void PrettyPrintRec(BinaryTreeNode<T> node, int spaces)
```

Comment:
Following to Single responsibility principle we should move that functionality to a new class.